docs/developing/install/steps.rst

```diff
@@ -117,6 +132,17 @@ You can set this is the admin interface.
    * A user to use as a supervisor (some researchers need a supervisor)
    * A user to use as secretary
 
+.. tip::
+   In a non-production enviroment it is advised to change the email in the settings.py file to
+        # E-mail settings
+        EMAIL_BACKEND = "django.core.mail.backends.filebased.EmailBackend"
+        EMAIL_FILE_PATH = "email/"
+        # EMAIL_HOST = "localhost"
+        # EMAIL_PORT = 2525
+        EMAIL_FROM = "T.D.Mees@uu.nl"
+        EMAIL_LOCAL_STAFF = "T.D.Mees@uu.nl"
+   Emails will now be send to a local directory instead of generating an error.
+
```

Comment:
Markup in this paragraph isn't exactly right. Take a look at the version that Github rendered here: https://github.com/DH-IT-Portal-Development/ethics/blob/f728ab7e9cd097159f651e13a315495ae9da0170/docs/developing/install/steps.rst

It appears you can't place a code block inside a tip, or at least not like this. Check out [the RST primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) for a quickstart on how to write RST.

It's better to build the docs locally and check everything is correct before committing and pushing. See the top-level README.md on how to build the docs. You can also use a tool like `restview` for live updates while editing RST.

Reply:
Second, I would recommend putting the above email suggestion in `debug_settings.py` rather than in `settings.py`. The former is included in the global `.gitignore` so that the changes in there never get pushed and don't get lost when switching branches.